src/lib/src/charts/timeseries-chart.component.ts

```diff
@@ -59,9 +65,21 @@ export class TimeseriesChartComponent implements OnChanges {
         legend: {
           show: false
         },
+        point: {
+          show: this.showPoints,
+          r: this.pointRadius,
+          focus: {
+            expand: {
+              r: this.pointRadiusOnHover
+            }
+          }
+        },
         color: {
           pattern: this.colors
         },
+        padding: {
+          right: this.paddingRight
```

Comment:
I also added paddingLeft at one point but found it was better to let C3 make those choice based on the length or magnitude of the values on the y axis or whatever it is doing. It's mostly padding right on category (category meaning non indexed chart values such as the dates) x axis that it doesn't seem to get right.